src/table/utils/selections-utils.ts

```diff
@@ -171,7 +171,7 @@ export const getMultiSelectedRows = (
     newSelectedRows[cell.qElemNumber] = cell.rowIdx;
     // also add the next or previous cell to selectedRows, based on which arrow is pressed
     const idxShift = evt.key === KeyCodes.DOWN ? 1 : -1;
-    const nextCell = pageRows[cell.pageRowIdx + idxShift][`col-${cell.colIdx}`] as Cell;
+    const nextCell = pageRows[cell.pageRowIdx + idxShift][`col-${cell.pageColIdx}`] as Cell;
```

Comment:
have you tried this with pagination?

Reply:
Yes see my other comment :) Not all that much but did test with a moved column, a non-moved column and using keyboard multiselect